src/components/escrowActions.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Flex, Spinner, Text } from '@chakra-ui/react';
+
+import { useUser } from '../contexts/UserContext';
+import { MinionService } from '../services/minionService';
+import { useOverlay } from '../contexts/OverlayContext';
```

Comment:
Check import order and react component body order:

https://daohaus.club/docs/devs/react-layount